src/tests/actions/userActions.test.js

```diff
@@ -89,4 +94,83 @@ describe('userAction', () => {
       },
     ]);
   });
+  it('should have social login initiated with google', async () => {
+    const user_data = {
+      user: {
+        access_token: 'token',
+      },
+    };
+    const type = [{ type: SOCIAL_LOGIN_INITIATED }];
+    googleLoginUser('/api/auth/google/', user_data)(store.dispatch);
+    await flushAllPromises();
+    expect(store.getActions()).toEqual(type);
+  });
+  it('should have social login initiated with facebook', async () => {
+    const user_data = {
+      user: {
+        access_token: 'token',
+      },
+    };
+    const type = [{ type: SOCIAL_LOGIN_INITIATED }];
+    facebookLoginUser('/api/auth/facebook/', user_data)(store.dispatch);
+    await flushAllPromises();
+    expect(store.getActions()).toEqual(type);
+  });
+  it('should have social login initiated with google', async () => {
+    const user_data = {
+      user: {
+        access_token: 'token',
+      },
+    };
+    const type = [{ type: SOCIAL_LOGIN_INITIATED }];
+    facebookLoginUser('/api/auth/google/', user_data)(store.dispatch);
+    await flushAllPromises();
+    expect(store.getActions()).toEqual(type);
+  });
+  it('should have social login successfull facebook', () => {
+    const response = {
+      user: {
+        access_token: 'token',
+      },
+    };
+    const user_data = {
+      user: {
+        access_token: 'token',
+      },
+    };
+    const expectedActions = [
+      { type: SOCIAL_LOGIN_INITIATED },
+      { type: SOCIAL_LOGIN_SUCCESS },
+    ];
+    mock
+      .onPost('/api/auth/google/', user_data)
+      .reply(200, response);
+    store.dispatch(googleLoginUser('/api/auth/google/', user_data))
+      .then(() => {
+        expect(store.getActions()).toEqual(expectedActions);
+      });
+  });
+  it('should have social login successfull facebook', () => {
```

Comment:
Please change the spelling here `login successfull` and the meaning of the entire statement.
The string passed here to `it` is the same as on line `130`